products/cert-manager.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Certmanager
+category: app
+tags: cncf certificate-management
+permalink: /cert-manager
+alternate_urls:
+-   /cert-manager
+releasePolicyLink: https://cert-manager.io/docs/releases/
+changelogTemplate: https://github.com/cert-manager/cert-manager/releases/tag/v__LATEST__
+releaseDateColumn: true
+eolColumn: Support
+
+auto:
+-   git: https://github.com/cert-manager/cert-manager.git
+
+identifiers:
+-   repology: cert-manager
+-   purl: pkg:docker/cert-manager/cert-manager
+
+releases:
+-   releaseCycle: "1.13"
+    releaseDate: 2023-09-12
+    eol: false
+    latest: "1.13.2"
+    latestReleaseDate: 2023-10-30
+
+-   releaseCycle: "1.12"
+    releaseDate: 2023-05-19
+    eol: false
+    latest: "1.12.6"
+    latestReleaseDate: 2023-10-30
+
+-   releaseCycle: "1.11"
+    releaseDate: 2023-01-11
+    eol: 2023-09-12
+    latest: "1.11.5"
+    latestReleaseDate: 2023-09-01
+
```

Comment:
On https://cert-manager.io/docs/releases/#old-releases are old releases back to 1.0. If you have the time please add the cycle entries too, thanks.

Reply:
Sure , can I address this in a follow up PR ?

Reply:
Yes, you can do that in a follow up PR -> solved